[0.9.3-rc2] panic: runtime error: makeslice: len out of range, persists on restart

I had a single-server InfluxDB instance up and running for ~20 minutes and then it crashed with the following panic.  Attempts to restart the instance resulted in a similar error (look for "InfluxDB" startup banner below).  Here is the output from the log:

```
[http] 2015/08/25 15:07:24 127.0.0.1 - - [25/Aug/2015:15:07:24 -0400] POST /write HTTP/1.1 204 0 - Apache-HttpClient/4.3.6 (java 1.5) 84ceda44-4b5c-11e5-840d-000000000000 22.0013ms
[http] 2015/08/25 15:07:24 127.0.0.1 - - [25/Aug/2015:15:07:24 -0400] POST /write HTTP/1.1 204 0 - Apache-HttpClient/4.3.6 (java 1.5) 852d4f57-4b5c-11e5-840e-000000000000 87.0049ms
[retention] 2015/08/25 15:07:31 retention policy enforcement check commencing
[retention] 2015/08/25 15:07:31 retention policy shard deletion check commencing
[http] 2015/08/25 15:07:31 127.0.0.1 - - [25/Aug/2015:15:07:31 -0400] POST /write HTTP/1.1 204 0 - Apache-HttpClient/4.3.6 (java 1.5) 892ee96b-4b5c-11e5-840f-000000000000 80.0046ms
[http] 2015/08/25 15:07:32 127.0.0.1 - - [25/Aug/2015:15:07:32 -0400] POST /write HTTP/1.1 204 0 - Apache-HttpClient/4.3.6 (java 1.5) 89f98f63-4b5c-11e5-8410-000000000000 21.0012ms
[http] 2015/08/25 15:07:33 127.0.0.1 - - [25/Aug/2015:15:07:33 -0400] POST /write HTTP/1.1 204 0 - Apache-HttpClient/4.3.6 (java 1.5) 8a13f596-4b5c-11e5-8411-000000000000 55.0031ms
[http] 2015/08/25 15:07:33 127.0.0.1 - - [25/Aug/2015:15:07:33 -0400] POST /write HTTP/1.1 204 0 - Apache-HttpClient/4.3.6 (java 1.5) 8a94e749-4b5c-11e5-8412-000000000000 32.0018ms
[http] 2015/08/25 15:07:34 127.0.0.1 - - [25/Aug/2015:15:07:34 -0400] POST /write HTTP/1.1 204 0 - Apache-HttpClient/4.3.6 (java 1.5) 8aaba3ee-4b5c-11e5-8413-000000000000 79.0045ms
[http] 2015/08/25 15:07:34 127.0.0.1 - - [25/Aug/2015:15:07:34 -0400] POST /write HTTP/1.1 204 0 - Apache-HttpClient/4.3.6 (java 1.5) 8ac5e310-4b5c-11e5-8414-000000000000 20.0012ms
[http] 2015/08/25 15:07:34 127.0.0.1 - - [25/Aug/2015:15:07:34 -0400] POST /write HTTP/1.1 204 0 - Apache-HttpClient/4.3.6 (java 1.5) 8b2346af-4b5c-11e5-8415-000000000000 77.0044ms
[http] 2015/08/25 15:07:36 127.0.0.1 - - [25/Aug/2015:15:07:36 -0400] GET /query?db=app_stats&q=select+connected%2C+busy%2C+decode%2C+%22read%22%2C+process%2C+rcvd_msgs_count%2C+rcvd_msgs_rate%2C+read_count%2C+read_events+from+%22fix.session.TNBT%2BSIM2%22 HTTP/1.1 200 2926 - InfluxDBShell/0.9.3-rc2 8be84741-4b5c-11e5-8416-000000000000 11.0006ms
panic: runtime error: makeslice: len out of range

goroutine 67 [running]:
github.com/influxdb/influxdb/tsdb/engine/wal.(*Log).readMetadataFile(0xc082158460, 0xc082222500, 0x47, 0x0, 0x0, 0x0, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tsdb/engine/wal/wal.go:371 +0x3af
github.com/influxdb/influxdb/tsdb/engine/wal.(*Log).flushMetadata(0xc082158460, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tsdb/engine/wal/wal.go:637 +0x207
github.com/influxdb/influxdb/tsdb/engine/wal.(*Log).autoflusher(0xc082158460, 0xc082128600)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tsdb/engine/wal/wal.go:612 +0x370
created by github.com/influxdb/influxdb/tsdb/engine/wal.(*Log).Open
	C:/projects/gocode/src/github.com/influxdb/influxdb/tsdb/engine/wal/wal.go:219 +0x7a2

goroutine 1 [chan receive, 20 minutes]:
main.(*Main).Run(0xc08200be40, 0xc082009fc0, 0x2, 0x2, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/cmd/influxd/main.go:91 +0x77e
main.main()
	C:/projects/gocode/src/github.com/influxdb/influxdb/cmd/influxd/main.go:42 +0xe3

goroutine 6 [syscall, 20 minutes]:
os/signal.loop()
	c:/go/src/os/signal/signal_unix.go:21 +0x26
created by os/signal.initÂ·1
	c:/go/src/os/signal/signal_unix.go:27 +0x3c

goroutine 17 [chan receive, 20 minutes]:
github.com/influxdb/influxdb/tcp.(*listener).Accept(0xc08202cd68, 0x0, 0x0, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tcp/mux.go:117 +0x87
github.com/influxdb/influxdb/meta.(*Store).serveExecListener(0xc082010280)
	C:/projects/gocode/src/github.com/influxdb/influxdb/meta/store.go:619 +0x96
created by github.com/influxdb/influxdb/meta.(*Store).Open
	C:/projects/gocode/src/github.com/influxdb/influxdb/meta/store.go:239 +0x398

goroutine 8 [IO wait, 20 minutes]:
net.(*pollDesc).Wait(0xc0820a02f0, 0x72, 0x0, 0x0)
	c:/go/src/net/fd_poll_runtime.go:84 +0x4e
net.(*ioSrv).ExecIO(0xc08202cd48, 0xc0820a01e0, 0xb321d0, 0x8, 0xc0820e4000, 0xc0820e2000, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:188 +0x305
net.(*netFD).acceptOne(0xc0820a0180, 0xc0820de000, 0x2, 0x2, 0xc0820a01e0, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:558 +0x547
net.(*netFD).accept(0xc0820a0180, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:585 +0x17a
net.(*TCPListener).AcceptTCP(0xc08202cd58, 0xb44630, 0x0, 0x0)
	c:/go/src/net/tcpsock_posix.go:234 +0x55
net.(*TCPListener).Accept(0xc08202cd58, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/tcpsock_posix.go:244 +0x53
github.com/influxdb/influxdb/tcp.(*Mux).Serve(0xc0820d4f30, 0x138718, 0xc08202cd58, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tcp/mux.go:45 +0x5e
created by github.com/influxdb/influxdb/cmd/influxd/run.funcÂ·003
	C:/projects/gocode/src/github.com/influxdb/influxdb/cmd/influxd/run/server.go:313 +0x8f9

goroutine 9 [chan receive, 20 minutes]:
github.com/influxdb/influxdb/tcp.(*listener).Accept(0xc08202cd60, 0x0, 0x0, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tcp/mux.go:117 +0x87
github.com/influxdb/influxdb/meta.(*raftLayer).Accept(0xc0820d4fc0, 0x0, 0x0, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/meta/store.go:2058 +0x6b
github.com/hashicorp/raft.(*NetworkTransport).listen(0xc08201b650)
	C:/projects/gocode/src/github.com/hashicorp/raft/net_transport.go:346 +0x57
created by github.com/hashicorp/raft.NewNetworkTransport
	C:/projects/gocode/src/github.com/hashicorp/raft/net_transport.go:138 +0x2cb

goroutine 10 [select]:
github.com/hashicorp/raft.(*Raft).leaderLoop(0xc0820b01c0)
	C:/projects/gocode/src/github.com/hashicorp/raft/raft.go:826 +0xc83
github.com/hashicorp/raft.(*Raft).runLeader(0xc0820b01c0)
	C:/projects/gocode/src/github.com/hashicorp/raft/raft.go:798 +0x755
github.com/hashicorp/raft.(*Raft).run(0xc0820b01c0)
	C:/projects/gocode/src/github.com/hashicorp/raft/raft.go:595 +0xd0
github.com/hashicorp/raft.*Raft.(github.com/hashicorp/raft.run)Â·fm()
	C:/projects/gocode/src/github.com/hashicorp/raft/raft.go:245 +0x2e
github.com/hashicorp/raft.funcÂ·011()
	C:/projects/gocode/src/github.com/hashicorp/raft/state.go:152 +0x58
created by github.com/hashicorp/raft.(*raftState).goFunc
	C:/projects/gocode/src/github.com/hashicorp/raft/state.go:153 +0xea

goroutine 11 [select, 19 minutes]:
github.com/hashicorp/raft.(*Raft).runFSM(0xc0820b01c0)
	C:/projects/gocode/src/github.com/hashicorp/raft/raft.go:502 +0x104a
github.com/hashicorp/raft.*Raft.(github.com/hashicorp/raft.runFSM)Â·fm()
	C:/projects/gocode/src/github.com/hashicorp/raft/raft.go:246 +0x2e
github.com/hashicorp/raft.funcÂ·011()
	C:/projects/gocode/src/github.com/hashicorp/raft/state.go:152 +0x58
created by github.com/hashicorp/raft.(*raftState).goFunc
	C:/projects/gocode/src/github.com/hashicorp/raft/state.go:153 +0xea

goroutine 12 [select]:
github.com/hashicorp/raft.(*Raft).runSnapshots(0xc0820b01c0)
	C:/projects/gocode/src/github.com/hashicorp/raft/raft.go:1613 +0x46d
github.com/hashicorp/raft.*Raft.(github.com/hashicorp/raft.runSnapshots)Â·fm()
	C:/projects/gocode/src/github.com/hashicorp/raft/raft.go:247 +0x2e
github.com/hashicorp/raft.funcÂ·011()
	C:/projects/gocode/src/github.com/hashicorp/raft/state.go:152 +0x58
created by github.com/hashicorp/raft.(*raftState).goFunc
	C:/projects/gocode/src/github.com/hashicorp/raft/state.go:153 +0xea

goroutine 13 [select, 20 minutes]:
github.com/influxdb/influxdb/meta.(*localRaft).logLeaderChanges(0xc08200f4a0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/meta/state.go:187 +0x475
created by github.com/influxdb/influxdb/meta.(*localRaft).open
	C:/projects/gocode/src/github.com/influxdb/influxdb/meta/state.go:177 +0x1313

goroutine 18 [chan receive, 20 minutes]:
github.com/influxdb/influxdb/tcp.(*listener).Accept(0xc08202cd70, 0x0, 0x0, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tcp/mux.go:117 +0x87
github.com/influxdb/influxdb/meta.(*Store).serveRPCListener(0xc082010280)
	C:/projects/gocode/src/github.com/influxdb/influxdb/meta/store.go:723 +0xd0
created by github.com/influxdb/influxdb/meta.(*Store).Open
	C:/projects/gocode/src/github.com/influxdb/influxdb/meta/store.go:242 +0x3e2

goroutine 15 [select]:
github.com/influxdb/influxdb/services/hh.(*Service).retryWrites(0xc08205f720)
	C:/projects/gocode/src/github.com/influxdb/influxdb/services/hh/service.go:103 +0x2ee
created by github.com/influxdb/influxdb/services/hh.(*Service).Open
	C:/projects/gocode/src/github.com/influxdb/influxdb/services/hh/service.go:67 +0x215

goroutine 20 [select, 20 minutes]:
github.com/influxdb/influxdb/cmd/influxd/run.(*Server).monitorErrorChan(0xc0820ac1c0, 0xc08200eba0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/cmd/influxd/run/server.go:442 +0x151
created by github.com/influxdb/influxdb/cmd/influxd/run.funcÂ·003
	C:/projects/gocode/src/github.com/influxdb/influxdb/cmd/influxd/run/server.go:319 +0xa6a

goroutine 16 [select, 20 minutes]:
github.com/influxdb/influxdb/services/hh.(*Service).expireWrites(0xc08205f720)
	C:/projects/gocode/src/github.com/influxdb/influxdb/services/hh/service.go:121 +0x2ab
created by github.com/influxdb/influxdb/services/hh.(*Service).Open
	C:/projects/gocode/src/github.com/influxdb/influxdb/services/hh/service.go:68 +0x22f

goroutine 33 [chan receive, 20 minutes]:
github.com/influxdb/influxdb/tcp.(*listener).Accept(0xc08202cd78, 0x0, 0x0, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tcp/mux.go:117 +0x87
github.com/influxdb/influxdb/cluster.(*Service).serve(0xc08201b180)
	C:/projects/gocode/src/github.com/influxdb/influxdb/cluster/service.go:83 +0x109
created by github.com/influxdb/influxdb/cluster.(*Service).Open
	C:/projects/gocode/src/github.com/influxdb/influxdb/cluster/service.go:60 +0x13f

goroutine 34 [select]:
github.com/influxdb/influxdb/services/precreator.(*Service).runPrecreation(0xc08200dae0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/services/precreator/service.go:75 +0x40e
created by github.com/influxdb/influxdb/services/precreator.(*Service).Open
	C:/projects/gocode/src/github.com/influxdb/influxdb/services/precreator/service.go:53 +0x211

goroutine 35 [chan receive, 20 minutes]:
github.com/influxdb/influxdb/tcp.(*listener).Accept(0xc08202cd80, 0x0, 0x0, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tcp/mux.go:117 +0x87
github.com/influxdb/influxdb/services/snapshotter.(*Service).serve(0xc08200dbd0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/services/snapshotter/service.go:75 +0xb4
created by github.com/influxdb/influxdb/services/snapshotter.(*Service).Open
	C:/projects/gocode/src/github.com/influxdb/influxdb/services/snapshotter/service.go:48 +0x13a

goroutine 36 [IO wait, 20 minutes]:
net.(*pollDesc).Wait(0xc0820e22f0, 0x72, 0x0, 0x0)
	c:/go/src/net/fd_poll_runtime.go:84 +0x4e
net.(*ioSrv).ExecIO(0xc08202cd48, 0xc0820e21e0, 0xb321d0, 0x8, 0xc082004a60, 0xc0820a0000, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:188 +0x305
net.(*netFD).acceptOne(0xc0820e2180, 0xc0820de0e0, 0x2, 0x2, 0xc0820e21e0, 0xc0820e0500, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:558 +0x547
net.(*netFD).accept(0xc0820e2180, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:585 +0x17a
net.(*TCPListener).AcceptTCP(0xc082134048, 0xc082106380, 0x0, 0x0)
	c:/go/src/net/tcpsock_posix.go:234 +0x55
net.(*TCPListener).Accept(0xc082134048, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/tcpsock_posix.go:244 +0x53
net/http.(*Server).Serve(0xc08210a0c0, 0x138718, 0xc082134048, 0x0, 0x0)
	c:/go/src/net/http/server.go:1728 +0x99
net/http.Serve(0x138718, 0xc082134048, 0x13d588, 0xc082004a50, 0x0, 0x0)
	c:/go/src/net/http/server.go:1606 +0xa8
github.com/influxdb/influxdb/services/admin.(*Service).serve(0xc08200dc70)
	C:/projects/gocode/src/github.com/influxdb/influxdb/services/admin/service.go:107 +0x14e
created by github.com/influxdb/influxdb/services/admin.(*Service).Open
	C:/projects/gocode/src/github.com/influxdb/influxdb/services/admin/service.go:70 +0x479

goroutine 37 [select]:
github.com/influxdb/influxdb/services/continuous_querier.(*Service).backgroundLoop(0xc08201b1f0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/services/continuous_querier/service.go:157 +0x3f0
created by github.com/influxdb/influxdb/services/continuous_querier.(*Service).Open
	C:/projects/gocode/src/github.com/influxdb/influxdb/services/continuous_querier/service.go:93 +0x2e6

goroutine 38 [IO wait]:
net.(*pollDesc).Wait(0xc0820e2470, 0x72, 0x0, 0x0)
	c:/go/src/net/fd_poll_runtime.go:84 +0x4e
net.(*ioSrv).ExecIO(0xc08202cd48, 0xc0820e2360, 0xb321d0, 0x8, 0xc0821fffe0, 0xc08258c480, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:188 +0x305
net.(*netFD).acceptOne(0xc0820e2300, 0xc0820de460, 0x2, 0x2, 0xc0820e2360, 0x127ae8, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:558 +0x547
net.(*netFD).accept(0xc0820e2300, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:585 +0x17a
net.(*TCPListener).AcceptTCP(0xc082134050, 0xc08216bdc8, 0x0, 0x0)
	c:/go/src/net/tcpsock_posix.go:234 +0x55
net.(*TCPListener).Accept(0xc082134050, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/tcpsock_posix.go:244 +0x53
net/http.(*Server).Serve(0xc082128420, 0x138718, 0xc082134050, 0x0, 0x0)
	c:/go/src/net/http/server.go:1728 +0x99
net/http.Serve(0x138718, 0xc082134050, 0x13d5b0, 0xc08201b260, 0x0, 0x0)
	c:/go/src/net/http/server.go:1606 +0xa8
github.com/influxdb/influxdb/services/httpd.(*Service).serve(0xc08200ddb0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/services/httpd/service.go:108 +0x80
created by github.com/influxdb/influxdb/services/httpd.(*Service).Open
	C:/projects/gocode/src/github.com/influxdb/influxdb/services/httpd/service.go:76 +0x692

goroutine 39 [select]:
github.com/influxdb/influxdb/services/retention.(*Service).deleteShardGroups(0xc08200f0e0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/services/retention/service.go:69 +0x349
created by github.com/influxdb/influxdb/services/retention.(*Service).Open
	C:/projects/gocode/src/github.com/influxdb/influxdb/services/retention/service.go:45 +0x13f

goroutine 40 [select]:
github.com/influxdb/influxdb/services/retention.(*Service).deleteShards(0xc08200f0e0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/services/retention/service.go:102 +0x722
created by github.com/influxdb/influxdb/services/retention.(*Service).Open
	C:/projects/gocode/src/github.com/influxdb/influxdb/services/retention/service.go:46 +0x159

goroutine 41 [chan receive, 20 minutes]:
github.com/influxdb/influxdb/cmd/influxd/run.(*Server).startServerReporting(0xc0820ac1c0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/cmd/influxd/run/server.go:393 +0x1de
created by github.com/influxdb/influxdb/cmd/influxd/run.funcÂ·003
	C:/projects/gocode/src/github.com/influxdb/influxdb/cmd/influxd/run/server.go:342 +0xf15

goroutine 42 [select, 20 minutes]:
github.com/influxdb/influxdb/cmd/influxd/run.(*Command).monitorServerErrors(0xc0820ae300)
	C:/projects/gocode/src/github.com/influxdb/influxdb/cmd/influxd/run/command.go:140 +0x2e1
created by github.com/influxdb/influxdb/cmd/influxd/run.(*Command).Run
	C:/projects/gocode/src/github.com/influxdb/influxdb/cmd/influxd/run/command.go:122 +0xdf6

goroutine 81 [IO wait, 18 minutes]:
net.(*pollDesc).Wait(0xc0820a42f0, 0x72, 0x0, 0x0)
	c:/go/src/net/fd_poll_runtime.go:84 +0x4e
net.(*ioSrv).ExecIO(0xc08202cd48, 0xc0820a41e0, 0xb13770, 0x7, 0xc7b108, 0xb14f70, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:188 +0x305
net.(*netFD).Read(0xc0820a4180, 0xc082156000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:470 +0x180
net.(*conn).Read(0xc08202c048, 0xc082156000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/net.go:121 +0xe3
net/http.(*liveSwitchReader).Read(0xc08205e048, 0xc082156000, 0x1000, 0x1000, 0xc082452450, 0x0, 0x0)
	c:/go/src/net/http/server.go:214 +0xb2
io.(*LimitedReader).Read(0xc0820e00a0, 0xc082156000, 0x1000, 0x1000, 0xc082106380, 0x0, 0x0)
	c:/go/src/io/io.go:408 +0xd5
bufio.(*Reader).fill(0xc0820d0420)
	c:/go/src/bufio/bufio.go:97 +0x1d5
bufio.(*Reader).ReadSlice(0xc0820d0420, 0x13910a, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:295 +0x25e
bufio.(*Reader).ReadLine(0xc0820d0420, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:324 +0x69
net/textproto.(*Reader).readLineSlice(0xc0823e87e0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:55 +0xa5
net/textproto.(*Reader).ReadLine(0xc0823e87e0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:36 +0x56
net/http.ReadRequest(0xc0820d0420, 0xc0821305b0, 0x0, 0x0)
	c:/go/src/net/http/request.go:598 +0xd2
net/http.(*conn).readRequest(0xc08205e000, 0x0, 0x0, 0x0)
	c:/go/src/net/http/server.go:586 +0x276
net/http.(*conn).serve(0xc08205e000)
	c:/go/src/net/http/server.go:1162 +0x6a5
created by net/http.(*Server).Serve
	c:/go/src/net/http/server.go:1751 +0x365

goroutine 82 [IO wait, 18 minutes]:
net.(*pollDesc).Wait(0xc0820a4470, 0x72, 0x0, 0x0)
	c:/go/src/net/fd_poll_runtime.go:84 +0x4e
net.(*ioSrv).ExecIO(0xc08202cd48, 0xc0820a4360, 0xb13770, 0x7, 0xc7b108, 0xb14f70, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:188 +0x305
net.(*netFD).Read(0xc0820a4300, 0xc08201d000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:470 +0x180
net.(*conn).Read(0xc08202c050, 0xc08201d000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/net.go:121 +0xe3
net/http.(*liveSwitchReader).Read(0xc08205e0e8, 0xc08201d000, 0x1000, 0x1000, 0xc082452240, 0x0, 0x0)
	c:/go/src/net/http/server.go:214 +0xb2
io.(*LimitedReader).Read(0xc0820e00e0, 0xc08201d000, 0x1000, 0x1000, 0xc08202ea80, 0x0, 0x0)
	c:/go/src/io/io.go:408 +0xd5
bufio.(*Reader).fill(0xc0820d06c0)
	c:/go/src/bufio/bufio.go:97 +0x1d5
bufio.(*Reader).ReadSlice(0xc0820d06c0, 0x13920a, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:295 +0x25e
bufio.(*Reader).ReadLine(0xc0820d06c0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:324 +0x69
net/textproto.(*Reader).readLineSlice(0xc0824541b0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:55 +0xa5
net/textproto.(*Reader).ReadLine(0xc0824541b0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:36 +0x56
net/http.ReadRequest(0xc0820d06c0, 0xc0821301a0, 0x0, 0x0)
	c:/go/src/net/http/request.go:598 +0xd2
net/http.(*conn).readRequest(0xc08205e0a0, 0x0, 0x0, 0x0)
	c:/go/src/net/http/server.go:586 +0x276
net/http.(*conn).serve(0xc08205e0a0)
	c:/go/src/net/http/server.go:1162 +0x6a5
created by net/http.(*Server).Serve
	c:/go/src/net/http/server.go:1751 +0x365

goroutine 83 [IO wait]:
net.(*pollDesc).Wait(0xc0820a45f0, 0x72, 0x0, 0x0)
	c:/go/src/net/fd_poll_runtime.go:84 +0x4e
net.(*ioSrv).ExecIO(0xc08202cd48, 0xc0820a44e0, 0xb13770, 0x7, 0xc7b108, 0xb14f70, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:188 +0x305
net.(*netFD).Read(0xc0820a4480, 0xc082171000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:470 +0x180
net.(*conn).Read(0xc08202c098, 0xc082171000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/net.go:121 +0xe3
net/http.(*liveSwitchReader).Read(0xc08205e188, 0xc082171000, 0x1000, 0x1000, 0xc0824e31e8, 0x0, 0x0)
	c:/go/src/net/http/server.go:214 +0xb2
io.(*LimitedReader).Read(0xc0820e0120, 0xc082171000, 0x1000, 0x1000, 0xc0821b0380, 0x0, 0x0)
	c:/go/src/io/io.go:408 +0xd5
bufio.(*Reader).fill(0xc0820d0780)
	c:/go/src/bufio/bufio.go:97 +0x1d5
bufio.(*Reader).ReadSlice(0xc0820d0780, 0x13930a, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:295 +0x25e
bufio.(*Reader).ReadLine(0xc0820d0780, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:324 +0x69
net/textproto.(*Reader).readLineSlice(0xc082750bd0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:55 +0xa5
net/textproto.(*Reader).ReadLine(0xc082750bd0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:36 +0x56
net/http.ReadRequest(0xc0820d0780, 0xc0820b75f0, 0x0, 0x0)
	c:/go/src/net/http/request.go:598 +0xd2
net/http.(*conn).readRequest(0xc08205e140, 0x0, 0x0, 0x0)
	c:/go/src/net/http/server.go:586 +0x276
net/http.(*conn).serve(0xc08205e140)
	c:/go/src/net/http/server.go:1162 +0x6a5
created by net/http.(*Server).Serve
	c:/go/src/net/http/server.go:1751 +0x365

goroutine 84 [IO wait, 7 minutes]:
net.(*pollDesc).Wait(0xc0820a4770, 0x72, 0x0, 0x0)
	c:/go/src/net/fd_poll_runtime.go:84 +0x4e
net.(*ioSrv).ExecIO(0xc08202cd48, 0xc0820a4660, 0xb13770, 0x7, 0xc7b108, 0xb14f70, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:188 +0x305
net.(*netFD).Read(0xc0820a4600, 0xc082087000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:470 +0x180
net.(*conn).Read(0xc08202c0a0, 0xc082087000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/net.go:121 +0xe3
net/http.(*liveSwitchReader).Read(0xc08205e228, 0xc082087000, 0x1000, 0x1000, 0xc08212eb38, 0x0, 0x0)
	c:/go/src/net/http/server.go:214 +0xb2
io.(*LimitedReader).Read(0xc0820e0160, 0xc082087000, 0x1000, 0x1000, 0xc082106000, 0x0, 0x0)
	c:/go/src/io/io.go:408 +0xd5
bufio.(*Reader).fill(0xc0820d07e0)
	c:/go/src/bufio/bufio.go:97 +0x1d5
bufio.(*Reader).ReadSlice(0xc0820d07e0, 0x13910a, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:295 +0x25e
bufio.(*Reader).ReadLine(0xc0820d07e0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:324 +0x69
net/textproto.(*Reader).readLineSlice(0xc0821d6450, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:55 +0xa5
net/textproto.(*Reader).ReadLine(0xc0821d6450, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:36 +0x56
net/http.ReadRequest(0xc0820d07e0, 0xc0821a8b60, 0x0, 0x0)
	c:/go/src/net/http/request.go:598 +0xd2
net/http.(*conn).readRequest(0xc08205e1e0, 0x0, 0x0, 0x0)
	c:/go/src/net/http/server.go:586 +0x276
net/http.(*conn).serve(0xc08205e1e0)
	c:/go/src/net/http/server.go:1162 +0x6a5
created by net/http.(*Server).Serve
	c:/go/src/net/http/server.go:1751 +0x365

goroutine 85 [IO wait, 7 minutes]:
net.(*pollDesc).Wait(0xc0820a48f0, 0x72, 0x0, 0x0)
	c:/go/src/net/fd_poll_runtime.go:84 +0x4e
net.(*ioSrv).ExecIO(0xc08202cd48, 0xc0820a47e0, 0xb13770, 0x7, 0xc7b108, 0xb14f70, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:188 +0x305
net.(*netFD).Read(0xc0820a4780, 0xc082065000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:470 +0x180
net.(*conn).Read(0xc08202c0a8, 0xc082065000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/net.go:121 +0xe3
net/http.(*liveSwitchReader).Read(0xc08205e2c8, 0xc082065000, 0x1000, 0x1000, 0xc0821243d0, 0x0, 0x0)
	c:/go/src/net/http/server.go:214 +0xb2
io.(*LimitedReader).Read(0xc0820e0220, 0xc082065000, 0x1000, 0x1000, 0xc08202f180, 0x0, 0x0)
	c:/go/src/io/io.go:408 +0xd5
bufio.(*Reader).fill(0xc0820d0840)
	c:/go/src/bufio/bufio.go:97 +0x1d5
bufio.(*Reader).ReadSlice(0xc0820d0840, 0x13900a, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:295 +0x25e
bufio.(*Reader).ReadLine(0xc0820d0840, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:324 +0x69
net/textproto.(*Reader).readLineSlice(0xc0821d7260, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:55 +0xa5
net/textproto.(*Reader).ReadLine(0xc0821d7260, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:36 +0x56
net/http.ReadRequest(0xc0820d0840, 0xc082532f70, 0x0, 0x0)
	c:/go/src/net/http/request.go:598 +0xd2
net/http.(*conn).readRequest(0xc08205e280, 0x0, 0x0, 0x0)
	c:/go/src/net/http/server.go:586 +0x276
net/http.(*conn).serve(0xc08205e280)
	c:/go/src/net/http/server.go:1162 +0x6a5
created by net/http.(*Server).Serve
	c:/go/src/net/http/server.go:1751 +0x365

goroutine 62 [IO wait, 7 minutes]:
net.(*pollDesc).Wait(0xc0822242f0, 0x72, 0x0, 0x0)
	c:/go/src/net/fd_poll_runtime.go:84 +0x4e
net.(*ioSrv).ExecIO(0xc08202cd48, 0xc0822241e0, 0xb13770, 0x7, 0xc7b108, 0xb14f70, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:188 +0x305
net.(*netFD).Read(0xc082224180, 0xc0820a3000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:470 +0x180
net.(*conn).Read(0xc082084040, 0xc0820a3000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/net.go:121 +0xe3
net/http.(*liveSwitchReader).Read(0xc0820f40e8, 0xc0820a3000, 0x1000, 0x1000, 0xc0820a8cf0, 0x0, 0x0)
	c:/go/src/net/http/server.go:214 +0xb2
io.(*LimitedReader).Read(0xc0821e2600, 0xc0820a3000, 0x1000, 0x1000, 0xc0820ee700, 0x0, 0x0)
	c:/go/src/io/io.go:408 +0xd5
bufio.(*Reader).fill(0xc08227e780)
	c:/go/src/bufio/bufio.go:97 +0x1d5
bufio.(*Reader).ReadSlice(0xc08227e780, 0x138e0a, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:295 +0x25e
bufio.(*Reader).ReadLine(0xc08227e780, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:324 +0x69
net/textproto.(*Reader).readLineSlice(0xc082585800, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:55 +0xa5
net/textproto.(*Reader).ReadLine(0xc082585800, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:36 +0x56
net/http.ReadRequest(0xc08227e780, 0xc082221930, 0x0, 0x0)
	c:/go/src/net/http/request.go:598 +0xd2
net/http.(*conn).readRequest(0xc0820f40a0, 0x0, 0x0, 0x0)
	c:/go/src/net/http/server.go:586 +0x276
net/http.(*conn).serve(0xc0820f40a0)
	c:/go/src/net/http/server.go:1162 +0x6a5
created by net/http.(*Server).Serve
	c:/go/src/net/http/server.go:1751 +0x365

goroutine 61 [IO wait, 18 minutes]:
net.(*pollDesc).Wait(0xc0820a4a70, 0x72, 0x0, 0x0)
	c:/go/src/net/fd_poll_runtime.go:84 +0x4e
net.(*ioSrv).ExecIO(0xc08202cd48, 0xc0820a4960, 0xb13770, 0x7, 0xc7b108, 0xb14f70, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:188 +0x305
net.(*netFD).Read(0xc0820a4900, 0xc08212d000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:470 +0x180
net.(*conn).Read(0xc082084038, 0xc08212d000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/net.go:121 +0xe3
net/http.(*liveSwitchReader).Read(0xc0820f4048, 0xc08212d000, 0x1000, 0x1000, 0xc082452190, 0x0, 0x0)
	c:/go/src/net/http/server.go:214 +0xb2
io.(*LimitedReader).Read(0xc0821e25a0, 0xc08212d000, 0x1000, 0x1000, 0xc082106380, 0x0, 0x0)
	c:/go/src/io/io.go:408 +0xd5
bufio.(*Reader).fill(0xc08227e720)
	c:/go/src/bufio/bufio.go:97 +0x1d5
bufio.(*Reader).ReadSlice(0xc08227e720, 0x138f0a, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:295 +0x25e
bufio.(*Reader).ReadLine(0xc08227e720, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:324 +0x69
net/textproto.(*Reader).readLineSlice(0xc082454090, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:55 +0xa5
net/textproto.(*Reader).ReadLine(0xc082454090, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:36 +0x56
net/http.ReadRequest(0xc08227e720, 0xc0821300d0, 0x0, 0x0)
	c:/go/src/net/http/request.go:598 +0xd2
net/http.(*conn).readRequest(0xc0820f4000, 0x0, 0x0, 0x0)
	c:/go/src/net/http/server.go:586 +0x276
net/http.(*conn).serve(0xc0820f4000)
	c:/go/src/net/http/server.go:1162 +0x6a5
created by net/http.(*Server).Serve
	c:/go/src/net/http/server.go:1751 +0x365

goroutine 152 [IO wait, 7 minutes]:
net.(*pollDesc).Wait(0xc0822248f0, 0x72, 0x0, 0x0)
	c:/go/src/net/fd_poll_runtime.go:84 +0x4e
net.(*ioSrv).ExecIO(0xc08202cd48, 0xc0822247e0, 0xb13770, 0x7, 0xc7b108, 0xb14f70, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:188 +0x305
net.(*netFD).Read(0xc082224780, 0xc082487000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:470 +0x180
net.(*conn).Read(0xc082084050, 0xc082487000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/net.go:121 +0xe3
net/http.(*liveSwitchReader).Read(0xc08205e7c8, 0xc082487000, 0x1000, 0x1000, 0xc0824353a0, 0x0, 0x0)
	c:/go/src/net/http/server.go:214 +0xb2
io.(*LimitedReader).Read(0xc08245a060, 0xc082487000, 0x1000, 0x1000, 0xc082400000, 0x0, 0x0)
	c:/go/src/io/io.go:408 +0xd5
bufio.(*Reader).fill(0xc082128720)
	c:/go/src/bufio/bufio.go:97 +0x1d5
bufio.(*Reader).ReadSlice(0xc082128720, 0x138d0a, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:295 +0x25e
bufio.(*Reader).ReadLine(0xc082128720, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:324 +0x69
net/textproto.(*Reader).readLineSlice(0xc08243b710, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:55 +0xa5
net/textproto.(*Reader).ReadLine(0xc08243b710, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:36 +0x56
net/http.ReadRequest(0xc082128720, 0xc082260a90, 0x0, 0x0)
	c:/go/src/net/http/request.go:598 +0xd2
net/http.(*conn).readRequest(0xc08205e780, 0x0, 0x0, 0x0)
	c:/go/src/net/http/server.go:586 +0x276
net/http.(*conn).serve(0xc08205e780)
	c:/go/src/net/http/server.go:1162 +0x6a5
created by net/http.(*Server).Serve
	c:/go/src/net/http/server.go:1751 +0x365

goroutine 56 [select]:
github.com/influxdb/influxdb/tsdb/engine/wal.(*Log).autoflusher(0xc0820ac0e0, 0xc08227e240)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tsdb/engine/wal/wal.go:600 +0x4da
created by github.com/influxdb/influxdb/tsdb/engine/wal.(*Log).Open
	C:/projects/gocode/src/github.com/influxdb/influxdb/tsdb/engine/wal/wal.go:219 +0x7a2

goroutine 151 [IO wait, 18 minutes]:
net.(*pollDesc).Wait(0xc082224470, 0x72, 0x0, 0x0)
	c:/go/src/net/fd_poll_runtime.go:84 +0x4e
net.(*ioSrv).ExecIO(0xc08202cd48, 0xc082224360, 0xb13770, 0x7, 0xc7b108, 0xb14f70, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:188 +0x305
net.(*netFD).Read(0xc082224300, 0xc08247f000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:470 +0x180
net.(*conn).Read(0xc082084048, 0xc08247f000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/net.go:121 +0xe3
net/http.(*liveSwitchReader).Read(0xc08205e408, 0xc08247f000, 0x1000, 0x1000, 0xc0824522f0, 0x0, 0x0)
	c:/go/src/net/http/server.go:214 +0xb2
io.(*LimitedReader).Read(0xc08245a000, 0xc08247f000, 0x1000, 0x1000, 0xc082106380, 0x0, 0x0)
	c:/go/src/io/io.go:408 +0xd5
bufio.(*Reader).fill(0xc082128660)
	c:/go/src/bufio/bufio.go:97 +0x1d5
bufio.(*Reader).ReadSlice(0xc082128660, 0x138e0a, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:295 +0x25e
bufio.(*Reader).ReadLine(0xc082128660, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:324 +0x69
net/textproto.(*Reader).readLineSlice(0xc08225e4e0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:55 +0xa5
net/textproto.(*Reader).ReadLine(0xc08225e4e0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:36 +0x56
net/http.ReadRequest(0xc082128660, 0xc082130410, 0x0, 0x0)
	c:/go/src/net/http/request.go:598 +0xd2
net/http.(*conn).readRequest(0xc08205e3c0, 0x0, 0x0, 0x0)
	c:/go/src/net/http/server.go:586 +0x276
net/http.(*conn).serve(0xc08205e3c0)
	c:/go/src/net/http/server.go:1162 +0x6a5
created by net/http.(*Server).Serve
	c:/go/src/net/http/server.go:1751 +0x365

goroutine 273 [IO wait, 18 minutes]:
net.(*pollDesc).Wait(0xc082224a70, 0x72, 0x0, 0x0)
	c:/go/src/net/fd_poll_runtime.go:84 +0x4e
net.(*ioSrv).ExecIO(0xc08202cd48, 0xc082224960, 0xb13770, 0x7, 0xc7b108, 0xb14f70, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:188 +0x305
net.(*netFD).Read(0xc082224900, 0xc08248d000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:470 +0x180
net.(*conn).Read(0xc082084000, 0xc08248d000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	c:/go/src/net/net.go:121 +0xe3
net/http.(*liveSwitchReader).Read(0xc0823fc048, 0xc08248d000, 0x1000, 0x1000, 0xc0824523a0, 0x0, 0x0)
	c:/go/src/net/http/server.go:214 +0xb2
io.(*LimitedReader).Read(0xc08228c000, 0xc08248d000, 0x1000, 0x1000, 0xc08202ea80, 0x0, 0x0)
	c:/go/src/io/io.go:408 +0xd5
bufio.(*Reader).fill(0xc082129d40)
	c:/go/src/bufio/bufio.go:97 +0x1d5
bufio.(*Reader).ReadSlice(0xc082129d40, 0x138c0a, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:295 +0x25e
bufio.(*Reader).ReadLine(0xc082129d40, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/bufio/bufio.go:324 +0x69
net/textproto.(*Reader).readLineSlice(0xc0823e8990, 0x0, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:55 +0xa5
net/textproto.(*Reader).ReadLine(0xc0823e8990, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/textproto/reader.go:36 +0x56
net/http.ReadRequest(0xc082129d40, 0xc0821304e0, 0x0, 0x0)
	c:/go/src/net/http/request.go:598 +0xd2
net/http.(*conn).readRequest(0xc0823fc000, 0x0, 0x0, 0x0)
	c:/go/src/net/http/server.go:586 +0x276
net/http.(*conn).serve(0xc0823fc000)
	c:/go/src/net/http/server.go:1162 +0x6a5
created by net/http.(*Server).Serve
	c:/go/src/net/http/server.go:1751 +0x365


============================== 08/25/2015 15:07:40

 8888888           .d888 888                   8888888b.  888888b.
   888            d88P"  888                   888  "Y88b 888  "88b
   888            888    888                   888    888 888  .88P
   888   88888b.  888888 888 888  888 888  888 888    888 8888888K.
   888   888 "88b 888    888 888  888  Y8bd8P' 888    888 888  "Y88b
   888   888  888 888    888 888  888   X88K   888    888 888    888
   888   888  888 888    888 Y88b 888 .d8""8b. 888  .d88P 888   d88P
 8888888 888  888 888    888  "Y88888 888  888 8888888P"  8888888P"

Using configuration at: C:/projects/TNBT/data/influx/audit.ini
[metastore] 2015/08/25 15:07:40 Using data dir: C:\projects\TNBT\data\influx\audit\meta
[metastore] 2015/08/25 15:07:40 Skipping cluster join: already member of cluster: nodeId=1 raftEnabled=true peers=[localhost:6002]
[store] 2015/08/25 15:07:40 Using data dir: C:\projects\TNBT\data\influx\audit\data
[metastore] 2015/08/25 15:07:40 Node at localhost:6002 [Follower]
panic: runtime error: makeslice: len out of range

goroutine 1 [running]:
github.com/influxdb/influxdb/tsdb/engine/wal.(*Log).readMetadataFile(0xc0820ac700, 0xc08200ce10, 0x47, 0x0, 0x0, 0x0, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tsdb/engine/wal/wal.go:371 +0x3af
github.com/influxdb/influxdb/tsdb/engine/wal.(*Log).LoadMetadataIndex(0xc0820ac700, 0xc0820ec0f0, 0xc0820d3a70, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tsdb/engine/wal/wal.go:281 +0x1c4
github.com/influxdb/influxdb/tsdb/engine/bz1.(*Engine).LoadMetadataIndex(0xc0820d5d40, 0xc0820ec0f0, 0xc0820d3a70, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tsdb/engine/bz1/bz1.go:197 +0x100
github.com/influxdb/influxdb/tsdb.funcÂ·011(0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tsdb/shard.go:103 +0x53a
github.com/influxdb/influxdb/tsdb.(*Shard).Open(0xc0820f67e0, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tsdb/shard.go:108 +0x65
github.com/influxdb/influxdb/tsdb.(*Store).loadShards(0xc0820128f0, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tsdb/store.go:257 +0xe27
github.com/influxdb/influxdb/tsdb.(*Store).Open(0xc0820128f0, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tsdb/store.go:288 +0x2f4
github.com/influxdb/influxdb/cmd/influxd/run.funcÂ·003(0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/cmd/influxd/run/server.go:325 +0xab2
github.com/influxdb/influxdb/cmd/influxd/run.(*Server).Open(0xc0820ac1c0, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/cmd/influxd/run/server.go:347 +0x65
github.com/influxdb/influxdb/cmd/influxd/run.(*Command).Run(0xc0820ae300, 0xc082009fc0, 0x2, 0x2, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/cmd/influxd/run/command.go:112 +0x985
main.(*Main).Run(0xc08200be40, 0xc082009fc0, 0x2, 0x2, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/cmd/influxd/main.go:81 +0x3ff
main.main()
	C:/projects/gocode/src/github.com/influxdb/influxdb/cmd/influxd/main.go:42 +0xe3

goroutine 6 [syscall]:
os/signal.loop()
	c:/go/src/os/signal/signal_unix.go:21 +0x26
created by os/signal.initÂ·1
	c:/go/src/os/signal/signal_unix.go:27 +0x3c

goroutine 8 [IO wait]:
net.(*pollDesc).Wait(0xc0820a62f0, 0x72, 0x0, 0x0)
	c:/go/src/net/fd_poll_runtime.go:84 +0x4e
net.(*ioSrv).ExecIO(0xc08202cd48, 0xc0820a61e0, 0xb321d0, 0x8, 0xc0820e2000, 0xc0820e0000, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:188 +0x305
net.(*netFD).acceptOne(0xc0820a6180, 0xc0820dc000, 0x2, 0x2, 0xc0820a61e0, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:558 +0x547
net.(*netFD).accept(0xc0820a6180, 0x0, 0x0, 0x0)
	c:/go/src/net/fd_windows.go:585 +0x17a
net.(*TCPListener).AcceptTCP(0xc08202cd58, 0xb44630, 0x0, 0x0)
	c:/go/src/net/tcpsock_posix.go:234 +0x55
net.(*TCPListener).Accept(0xc08202cd58, 0x0, 0x0, 0x0, 0x0)
	c:/go/src/net/tcpsock_posix.go:244 +0x53
github.com/influxdb/influxdb/tcp.(*Mux).Serve(0xc0820d2f30, 0x139620, 0xc08202cd58, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tcp/mux.go:45 +0x5e
created by github.com/influxdb/influxdb/cmd/influxd/run.funcÂ·003
	C:/projects/gocode/src/github.com/influxdb/influxdb/cmd/influxd/run/server.go:313 +0x8f9

goroutine 9 [chan receive]:
github.com/influxdb/influxdb/tcp.(*listener).Accept(0xc08202cd60, 0x0, 0x0, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tcp/mux.go:117 +0x87
github.com/influxdb/influxdb/meta.(*raftLayer).Accept(0xc0820d3050, 0x0, 0x0, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/meta/store.go:2058 +0x6b
github.com/hashicorp/raft.(*NetworkTransport).listen(0xc08201b650)
	C:/projects/gocode/src/github.com/hashicorp/raft/net_transport.go:346 +0x57
created by github.com/hashicorp/raft.NewNetworkTransport
	C:/projects/gocode/src/github.com/hashicorp/raft/net_transport.go:138 +0x2cb

goroutine 10 [select]:
github.com/hashicorp/raft.(*Raft).runFollower(0xc0820b01c0)
	C:/projects/gocode/src/github.com/hashicorp/raft/raft.go:606 +0x8a7
github.com/hashicorp/raft.(*Raft).run(0xc0820b01c0)
	C:/projects/gocode/src/github.com/hashicorp/raft/raft.go:591 +0xa3
github.com/hashicorp/raft.*Raft.(github.com/hashicorp/raft.run)Â·fm()
	C:/projects/gocode/src/github.com/hashicorp/raft/raft.go:245 +0x2e
github.com/hashicorp/raft.funcÂ·011()
	C:/projects/gocode/src/github.com/hashicorp/raft/state.go:152 +0x58
created by github.com/hashicorp/raft.(*raftState).goFunc
	C:/projects/gocode/src/github.com/hashicorp/raft/state.go:153 +0xea

goroutine 11 [select]:
github.com/hashicorp/raft.(*Raft).runFSM(0xc0820b01c0)
	C:/projects/gocode/src/github.com/hashicorp/raft/raft.go:502 +0x104a
github.com/hashicorp/raft.*Raft.(github.com/hashicorp/raft.runFSM)Â·fm()
	C:/projects/gocode/src/github.com/hashicorp/raft/raft.go:246 +0x2e
github.com/hashicorp/raft.funcÂ·011()
	C:/projects/gocode/src/github.com/hashicorp/raft/state.go:152 +0x58
created by github.com/hashicorp/raft.(*raftState).goFunc
	C:/projects/gocode/src/github.com/hashicorp/raft/state.go:153 +0xea

goroutine 12 [select]:
github.com/hashicorp/raft.(*Raft).runSnapshots(0xc0820b01c0)
	C:/projects/gocode/src/github.com/hashicorp/raft/raft.go:1613 +0x46d
github.com/hashicorp/raft.*Raft.(github.com/hashicorp/raft.runSnapshots)Â·fm()
	C:/projects/gocode/src/github.com/hashicorp/raft/raft.go:247 +0x2e
github.com/hashicorp/raft.funcÂ·011()
	C:/projects/gocode/src/github.com/hashicorp/raft/state.go:152 +0x58
created by github.com/hashicorp/raft.(*raftState).goFunc
	C:/projects/gocode/src/github.com/hashicorp/raft/state.go:153 +0xea

goroutine 13 [select]:
github.com/influxdb/influxdb/meta.(*localRaft).logLeaderChanges(0xc08200f620)
	C:/projects/gocode/src/github.com/influxdb/influxdb/meta/state.go:187 +0x475
created by github.com/influxdb/influxdb/meta.(*localRaft).open
	C:/projects/gocode/src/github.com/influxdb/influxdb/meta/state.go:177 +0x1313

goroutine 14 [chan receive]:
github.com/influxdb/influxdb/tcp.(*listener).Accept(0xc08202cd68, 0x0, 0x0, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tcp/mux.go:117 +0x87
github.com/influxdb/influxdb/meta.(*Store).serveExecListener(0xc0820103c0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/meta/store.go:619 +0x96
created by github.com/influxdb/influxdb/meta.(*Store).Open
	C:/projects/gocode/src/github.com/influxdb/influxdb/meta/store.go:239 +0x398

goroutine 15 [chan receive]:
github.com/influxdb/influxdb/tcp.(*listener).Accept(0xc08202cd70, 0x0, 0x0, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/tcp/mux.go:117 +0x87
github.com/influxdb/influxdb/meta.(*Store).serveRPCListener(0xc0820103c0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/meta/store.go:723 +0xd0
created by github.com/influxdb/influxdb/meta.(*Store).Open
	C:/projects/gocode/src/github.com/influxdb/influxdb/meta/store.go:242 +0x3e2

goroutine 16 [select]:
github.com/influxdb/influxdb/meta.(*Store).WaitForLeader(0xc0820103c0, 0x0, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/meta/store.go:553 +0x416
github.com/influxdb/influxdb/meta.funcÂ·006(0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/meta/store.go:267 +0x5f
github.com/influxdb/influxdb/meta.(*Store).syncNodeInfo(0xc0820103c0, 0x0, 0x0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/meta/store.go:290 +0x8f
created by github.com/influxdb/influxdb/meta.(*Store).Open
	C:/projects/gocode/src/github.com/influxdb/influxdb/meta/store.go:253 +0x576

goroutine 33 [select]:
github.com/influxdb/influxdb/cmd/influxd/run.(*Server).monitorErrorChan(0xc0820ac1c0, 0xc08200eba0)
	C:/projects/gocode/src/github.com/influxdb/influxdb/cmd/influxd/run/server.go:442 +0x151
created by github.com/influxdb/influxdb/cmd/influxd/run.funcÂ·003
	C:/projects/gocode/src/github.com/influxdb/influxdb/cmd/influxd/run/server.go:319 +0xa6a
```

--- a/tsdb/engine/wal/wal.go
+++ b/tsdb/engine/wal/wal.go
@@ -325,13 +325,13 @@ func (l *Log) LoadMetadataIndex(index *tsdb.DatabaseIndex, measurementFields map
 // is meant to be called by bz1 BEFORE it updates its own index, since the metadata
 // is flushed here first.
 func (l *Log) DeleteSeries(keys []string) error {
-	// we want to stop any writes from happening to ensure the data gets cleared
-	l.mu.Lock()
-	defer l.mu.Unlock()
-
 	if err := l.flushMetadata(); err != nil {
 		return err
 	}
+
+	// we want to stop any writes from happening to ensure the data gets cleared
+	l.mu.Lock()
+	defer l.mu.Unlock()
 
 	for _, p := range l.partitions {
 		p.deleteSeries(keys)
@@ -435,6 +435,9 @@ func (l *Log) writeSeriesAndFields(fields map[string]*tsdb.MeasurementFields, se
 // nextMetaFile will close the current file if there is one open and open a new file to log
 // metadata updates to. This function assumes that you've locked l.mu elsewhere.
 func (l *Log) nextMetaFile() error {
+	l.mu.Lock()
+	defer l.mu.Unlock()
+
 	if l.metaFile != nil {
 		if err := l.metaFile.Close(); err != nil {
 			return err
@@ -455,7 +458,7 @@ func (l *Log) nextMetaFile() error {
 			return err
 		}
 
-		id = int(n)
+		id = int(n) + 1
 	}
 
 	nextFileName := filepath.Join(l.path, fmt.Sprintf("%06d.%s", id, MetaFileExtension))
@@ -620,6 +623,28 @@ func (l *Log) autoflusher(closing chan struct{}) {
 // metadata from previous files to the index. After a sucessful write, the metadata files
 // will be removed. While the flush to index is happening we aren't blocked for new metadata writes.
 func (l *Log) flushMetadata() error {
+	// make sure there's actually something in the metadata file to flush
+	size, err := func() (int64, error) {
+		l.mu.Lock()
+		defer l.mu.Unlock()
+
+		if l.metaFile == nil {
+			return 0, nil
+		}
+		st, err := l.metaFile.Stat()
+		if err != nil {
+			return 0, err
+		}
+
+		return st.Size(), nil
+	}()
+	if err != nil {
+		return err
+	} else if size == 0 {
+		return nil
+	}
+
+	// we have data, get a list of the existing files and rotate to a new one, then flush
 	files, err := l.metadataFiles()
 	if err != nil {
 		return err

